app/helpers/user_helper.rb

```diff
@@ -0,0 +1,3 @@
+def current_user
+  @user = User.find_by(id: session[:id])
```

Comment:
Watch out here: `@user` may be in a name collision with other instance variables. Best to follow Rails convention and name this instance variable `@current_user`. Also [memoize](https://www.justinweiss.com/articles/4-simple-memoization-patterns-in-ruby-and-one-gem/) this method, since it could be called many times in any one http request/response cycle:
```ruby
def current_user
  @current_user ||= User.find_by(id: session[:id])
end
```